website/docs/reference/resource-configs/bigquery-configs.md

```diff
@@ -718,3 +718,189 @@ Views with this configuration will be able to select from objects in `project_1.
 
 The `grant_access_to` config is not thread-safe when multiple views need to be authorized for the same dataset. The initial `dbt run` operation after a new `grant_access_to` config is added should therefore be executed in a single thread. Subsequent runs using the same configuration will not attempt to re-apply existing access grants, and can make use of multiple threads.
 
+<VersionBlock firstVersion="1.7">
+
+## Materialized views
+
+The BigQuery adapter supports [materialized views](https://cloud.google.com/bigquery/docs/materialized-views-intro)
+with the following configuration parameters:
+
+| Parameter                                              | Type                   | Required | Default | Change Monitoring Support |
+|--------------------------------------------------------|------------------------|----------|---------|---------------------------|
+| `on_configuration_change`                              | `<string>`             | no       | `apply` | n/a                       |
+| [`cluster_by`](#clustering-clause)                     | `[<string>]`           | no       | `none`  | drop/create               |
+| [`partition_by`](#partition-clause)                    | `{<dictionary>}`       | no       | `none`  | drop/create               |
+| [`enable_refresh`](#auto-refresh)                      | `<boolean>`            | no       | `true`  | alter                     |
+| [`refresh_interval_minutes`](#auto-refresh)            | `<float>`              | no       | `30`    | alter                     |
+| [`max_staleness`](#auto-refresh) (in Preview)          | `<interval>`           | no       | `none`  | alter                     |
+| `description`                                          | `<string>`             | no       | `none`  | alter                     |
+| [`labels`](#specifying-labels)                         | `{<string>: <string>}` | no       | `none`  | alter                     |
+| [`hours_to_expiration`](#controlling-table-expiration) | `<integer>`            | no       | `none`  | alter                     |
+| [`kms_key_name`](#using-kms-encryption)                | `<string>`             | no       | `none`  | alter                     |
+
+<Tabs
+  groupId="config-languages"
+  defaultValue="project-yaml"
+  values={[
+    { label: 'Project file', value: 'project-yaml', },
+    { label: 'Property file', value: 'property-yaml', },
+    { label: 'Config block', value: 'config', },
+  ]
+}>
+
+
+<TabItem value="project-yaml">
+
+<File name='dbt_project.yml'>
+directly to the materialized view in place.
```

Comment:
Maybe this can just be deleted altogether?

```suggestion
```

It looks like a rogue line that is causing some funky rendering:

<img width="350" alt="image" src="https://github.com/dbt-labs/docs.getdbt.com/assets/44704949/69af75fa-be7d-465e-b2ad-f29c48a200b5">